homeassistant/components/freebox/const.py

```diff
@@ -58,16 +66,41 @@
 ATTR_DETECTION = "detection"
 
 
+class Freeboxlabel(StrEnum):
+    """Available Freebox label."""
```

Comment:
Label ???

At what it refers ?

Reply:
To use enum values not hardcoded string

Reply:
If this is an issue, this should be solved in a separate PR, not an all-in-one PR